.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: go
```

Comment:
@src-d/applications data-retrieval team has a separate job for osx https://github.com/src-d/gitcollector/blob/5d60f873b2a9c74de7cc4544692ea3d85d09092e/.travis.yml#L37 
Is it optional?

Reply:
we also usually test on macOs separately. And Windows as well.

Reply:
@kyrcha let's do it in a separate PR

Reply:
@lwsanty Just pushed

Reply:
I am removing windows since it does not work out of the box

Reply:
It would be great if you could open an issue 🙏